clients/java/src/main/java/io/camunda/zeebe/client/impl/response/DecisionRequirementsImpl.java

```diff
@@ -26,27 +27,58 @@ public final class DecisionRequirementsImpl implements DecisionRequirements {
   private final int version;
   private final long decisionRequirementsKey;
   private final String resourceName;
+  private final String tenantId;
 
   public DecisionRequirementsImpl(final DecisionRequirementsMetadata metadata) {
     this(
         metadata.getDmnDecisionRequirementsId(),
         metadata.getDmnDecisionRequirementsName(),
         metadata.getVersion(),
         metadata.getDecisionRequirementsKey(),
-        metadata.getResourceName());
+        metadata.getResourceName(),
+        metadata.getTenantId());
   }
 
+  /**
+   * A constructor that provides an instance with an empty String for a tenantId.
```

Comment:
🔧 This comment is no longer accurate now that we pick the default tenant id.

Reply:
Woops, nice catch!